packages/zfp/meta.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: zfp
+  version: 1.0.1
+
+source:
+  # There's no sdist for zfpy on PyPI, so we get the tarball from the GitHub release instead
+  url: https://github.com/LLNL/zfp/archive/refs/tags/1.0.1.tar.gz
+  sha256: 4984db6a55bc919831966dd17ba5e47ca7ac58668f4fd278ebd98cd2200da66f
+
+build:
+  # cflags: -fexceptions
+  # cxxflags: -fexceptions
+  # ldflags: -fexceptions
+  type: static_library
+  exports: requested
+  script: |
+    mkdir -p pyodide_build
+    cd pyodide_build
+    emcmake cmake ${CMAKE_ARGS}                   \
+      -DBUILD_CFP=ON                              \
+      -DBUILD_UTILITIES=OFF                       \
+      -DBUILD_TESTING=ON                         \
```

Comment:
Do we need to build test?

Reply:
Thanks for the catch, no need to – I was just testing something :)

Reply:
Addressed in 3a99b33e3a2bdb448540e70400a14dd0e53587f4.